frontend/src/components/campAdmin/DialogCategoryCreate.vue

```diff
@@ -23,10 +104,17 @@ import { categoryRoute } from '@/router.js'
 import DialogForm from '@/components/dialog/DialogForm.vue'
 import DialogBase from '@/components/dialog/DialogBase.vue'
 import DialogCategoryForm from './DialogCategoryForm.vue'
+import PopoverPrompt from '../prompt/PopoverPrompt.vue'
+import router from '../../router.js'
```

Comment:
Shouldn't this be an absolute import?

Reply:
Fixed in https://github.com/ecamp/ecamp3/pull/4823/commits/7fe7f5f83c3ba1befa38bda774505325af4390ad